PowerEditor/installer/nativeLang/croatian.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!--
 	Croatian localization for Notepad++
-	Updated: June 25, 2025 by Elvis Gambiraža (el.gambo@gmail.com)
+	Updated: August 5, 2025 by Elvis Gambiraža (el.gambo@gmail.com)
 	All the comments are for explanation, they are not for translation.
 -->
 <NotepadPlus>
-	<Native-Langue name="Hrvatski" filename="croatian.xml" version="8.8.1">
+	<Native-Langue name="Hrvatski" filename="croatian.xml" version="8.8.4">
```

Comment:
For the PR of translation, don't guess or use the next version number. Use the current version number instead.
ref: https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests